source/style-guide/information-types/prototypes/reference.txt

```diff
@@ -27,6 +27,31 @@ contents, often called a short description. The short description
 explains a little about how the reference material is used, and may link
 to a concept or task page.
 
+Compatibility 
+-------------
+
+Optional. 
+
+A list of MongoDB deployment types that support the referenced method, operator, 
+or database command: 
+
+- `MongoDB Atlas <https://docs.atlas.mongodb.com/>`_: The fully managed service 
+  for MongoDB deployments in the cloud
+
+.. note:: 
+
+   If the command is supported on Atlas, specify whether it's available in all 
+   clusters, has limited support for some clusters, or is not available in 
+   specific clusters. If there are limitations to which clusters support the 
```

Comment:
[Optional suggestion]

It might help to super briefly mention a concrete example here, like a command that can only be run on tier M10 or higher.

Reply:
Ah that's a great suggestion! I'll link to an example once https://github.com/10gen/docs-mongodb-internal/pull/5403 is published!

Reply:
Linked users to the `aggregate` command's Compatibility section!